### PR TITLE
Fix break when columns count was not a divisor of the frames count

### PR DESCRIPTION
**Pull request by Alexander-Ploskin**
UI: [imgur](https://imgur.com/a/sx1mLa3)

### CODE REVIEW

File: src/main/kotlin/sliv/tool/scene/view/FrameView.kt [hunk 1]
```diff
@@ -41,11 +44,19 @@ class FrameView(
...
+    fun setFrame(frame: VisualizationFrame?) {
         currentJob?.cancel()
         landmarksViews = null
         image = null
 
+        clearLandmarks()
+        clearImage()
+
+        if (frame == null) {
+            clearImage()
```
Comment (DmiitriyJarosh) [inline src/main/kotlin/sliv/tool/scene/view/FrameView.kt]: you have already cleared image before

File: src/main/kotlin/sliv/tool/scene/view/SceneView.kt [hunk 2]
```diff
@@ -50,9 +50,11 @@ class SceneView : View() {
         val scaleProperty = SimpleDoubleProperty(1.0)
 
         val columnsNumber = 30 //TODO: should be set from the UI
+        val emptyFrames = (0 until (30 - scene.frames.count() % columnsNumber)).map { null }
```
Comment (DmiitriyJarosh) [inline src/main/kotlin/sliv/tool/scene/view/SceneView.kt]: `30` hardcoded

**Review by DmiitriyJarosh — CHANGES_REQUESTED** [review]

File: src/main/kotlin/sliv/tool/scene/view/SceneView.kt [hunk 3]
```diff
@@ -50,9 +50,11 @@ class SceneView : View() {
         val scaleProperty = SimpleDoubleProperty(1.0)
 
         val columnsNumber = 30 //TODO: should be set from the UI
+        val emptyFrames = (0 until (columnsNumber - scene.frames.count() % columnsNumber)).map { null }
```
Comment (DmiitriyJarosh) [inline src/main/kotlin/sliv/tool/scene/view/SceneView.kt:53]: can we use `repeat` here?

**Review by DmiitriyJarosh — APPROVED** [review]

File: src/main/kotlin/sliv/tool/scene/view/SceneView.kt (hunk 3 above)
Comment (Alexander-Ploskin) [reply src/main/kotlin/sliv/tool/scene/view/SceneView.kt:53]: I don't think, how it could look better

**Pull request merged**